pkg/metadata/metadata.go

```diff
@@ -110,3 +115,20 @@ func (manager *nodeMetadataManager) GetRegion() string {
 func (manager *nodeMetadataManager) GetWorkerID() string {
 	return manager.workerID
 }
+
+// fetchInstanceID fetches instance ID of the node from the node provider ID.
+func fetchInstanceID(providerID string) string {
+	// ProviderID looks like: ibm://c468d88aqwerytytytfe0f379bf9///qwertyvvnrja55g/<instance-id>
```

Comment:
Guna can we remove this this, and just keep 
// ProviderID looks like: ibm://<account-id>///<clusterID>/<Instance-ID>